search/typo_suggest_engine.go

```diff
@@ -13,9 +14,58 @@ import (
 	null "gopkg.in/volatiletech/null.v6"
 )
 
+type QueryFormatter struct {
+	pattern string
+	sources map[int]string
```

Comment:
Maybe we should give this field another name e.g. numericTerms

Reply:
Then it might be better
excledeTrems,
since the pattern can be anything

Reply:
excludeTerms